.github/workflows/dubbo_integration_test.yml

```diff
@@ -69,32 +69,129 @@ jobs:
     needs: [download-midwares-and-cache, build-agent-and-cache]
     strategy:
       matrix:
-        dubbo-version: ['2-5','2-6','2-7']
-        dubbo-versions: ['0','1','2','3','4','5','6','7','8','9','10','11','12']
+        registry-model: ['interface']
+        dubbo-version: ['2-5','2-6','2-7','3-0','3-1','3-2']
+        dubbo-versions: ['0','3','6','9','12']
```

Comment:
check your workflow result，3.0 3.1 3.2 run nothing for interface

Reply:
fixed